src/util/get-update-command.ts

```diff
@@ -24,10 +24,76 @@ async function isYarn(): Promise<boolean> {
   return !('_id' in pkg);
 }
 
+async function getConfigPrefix() {
+  const paths = [
+    process.env.npm_config_userconfig || process.env.NPM_CONFIG_USERCONFIG,
```

Comment:
Are these uppercase env vars actually a thing? I've never seen npm set uppercase env vars.

Reply:
They were always displayed with `$NPM_CONFIG_USERCONFIG` instead of `$npm_config_userconfig`, so I just want to be sure here

Reply:
Sounds good 👍